pkg/util/jaeger.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	jaeger "github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+)
+
+type nopCloser struct {
+}
+
+func (nopCloser) Close() error { return nil }
+
+// TraceInit registers Jaeger as the OpenTracing implementation.
+func TraceInit(serviceName string) io.Closer {
```

Comment:
It'd be nice if this matched the `Config` / `Run()` / `Stop()` pattern we use elsewhere in Cortex.

Reply:
I'll change it to follow the pattern used elsewhere, `TraceInit`, though, returns an `io.Closer` so `Close()` needs to be called instead of `Stop()`.